.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
```

Comment:
Please rename the master to main please!

I am surprised the repo was created with the non inklusive and offensive term master

Reply:
changed 👍